From: Allen Luce <allen.luce@prometheanworld.com>
Date: Thu, 18 Nov 2021 21:37:04 +0000
Subject: [PATCH] Force package name to org.chromium.chrome

This is so the package will match the list set in the core OS that
determines window settings:

https://gerrit-hk.waq.be/plugins/gitiles/AP9/platform/frameworks/base/+/refs/heads/master/core/res/res/values/config.xml#31

At some point this method should be changed to instead allow the
AndroidManifest.xml to determine the windowing state of the app.

Done for AP9-2571
---
 chrome/android/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -47,7 +47,7 @@ chrome_jni_registration_header =
 chrome_jni_for_test_registration_header = "$root_build_dir/gen/chrome/browser/android/chrome_jni_for_test_registration.h"

 if (current_toolchain == default_toolchain) {
-  _default_package = "org.bromite.bromite"
+  _default_package = "org.chromium.chrome"
   if (use_stable_package_name_for_trichrome) {
     _default_package += ".stable"
   } else if (android_channel != "default" && android_channel != "stable") {
-- 
2.17.1

